src/components/navbar/Navbar.js

```diff
@@ -1,22 +1,81 @@
 import React from 'react';
 import { translate } from 'react-i18next';
 import 'i18n/i18n';
+import { slide as Menu } from 'react-burger-menu';
 import styles from './navbar.scss';
 
-const Navbar = ({ t }) => (
-  <nav className={styles.navigationBar}>
-    <div className={styles.inner}>
-      <a href="#about" className={styles.navLinks}>
-        {t('About')}
-      </a>
-      <a href="#mission" className={styles.navLinks}>
-        {t('Mission')}
-      </a>
-      <a href="#contact" className={styles.navLinks}>
-        {t('Contact')}
-      </a>
+const Navbar = ({ t }) => {
+  const BurgerMenustyles = {
+    bmBurgerButton: {
+      position: 'fixed',
+      width: '36px',
+      height: '30px',
+      left: '36px',
+      top: '36px',
+    },
+    bmBurgerBars: {
+      background: '#21785b',
+    },
+    bmCrossButton: {
+      height: '24px',
+      width: '24px',
+    },
+    bmCross: {
+      background: '#979797',
+    },
+    bmMenu: {
+      background: '#afe9d6',
+      padding: '2.5em 1.5em 0',
+      fontSize: '1.15em',
+    },
+    bmMorphShape: {
+      fill: '#373a47',
+    },
+    bmItemList: {
+      color: '#b8b7ad',
+      padding: '0.8em',
+    },
+    bmItem: {
+      display: 'block',
+      padding: '1.2em',
+    },
+    bmOverlay: {
+      background: 'rgba(0, 0, 0, 0.3)',
+    },
+  };
+  const renderDesktop = (
+    <nav className={styles.navigationBar}>
+      <div className={styles.inner}>
+        <a href="#about" className={styles.navLinks}>
+          {t('About')}
+        </a>
+        <a href="#mission" className={styles.navLinks}>
+          {t('Mission')}
+        </a>
+        <a href="#contact" className={styles.navLinks}>
+          {t('Contact')}
+        </a>
+      </div>
+    </nav>
+  );
+  const renderMobile = (
+    <div>
+      <Menu styles={BurgerMenustyles} isOpen={false}>
+        <a href="#about" className={styles.navLinks}>
```

Comment:
Can you eliminate the redundancy? just put the links in a variable and use it in both places.

Reply:
I tried that already but due to some reason, the _Menu_ component isn't appending classes to the required child. Hence decided to keep it as it is.